content/en/resources/2024-07-30-demystifying-llms.md

```diff
@@ -1,15 +1,16 @@
 ---
-title: "Demystifying Llms"
-slug: "demystifying-llms"
+title: "Demystifying Hugging Face Models & How to Leverage Them For Business Impact"
+slug: "demystifying-hugging-face-models"
+aliases: /demystifying-llms
```

Comment:
I thought that changing the `slug` to include Hugging Face would be more SEO friendly. I don't believe the link to this page was shared much, but including this alias will ensure that there won't be any broken links.

Reply:
thanks for making this change!